tests/sys_arduino/arduino-test.sketch

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2018 Federico Pellegrin <fede@evolware.org>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for arduino module
+ *
+ * @author      Federico Pellegrin <fede@evolware.org>
+ *
+ * @}
+ */
+
+#include <stdlib.h>
+#include <string.h>
+
+#ifdef STDIO_UART_BAUDRATE
+#define SERIAL_BAUDRATE STDIO_UART_BAUDRATE
+#else
+#define SERIAL_BAUDRATE 115200
+#endif
+
+/* Input buffer for receiving chars on the serial port */
+char buf[64];
+
+/* Counter of received chars in currently receiving line */
+int count = 0;
+
+void setup(void)
+{
+    Serial.begin(SERIAL_BAUDRATE);
+    Serial.println("Hello Arduino!");
+}
+
+void loop(void)
+{
+    /* Read chars if available and seek for CR or LF */
+    while (Serial.available() > 0) {
+        int tmp = Serial.read();
+        if (tmp == '\n' || tmp == '\r') {
+           buf[count] = 0;
+
+           if (count > 1) {
+                if (strncmp(buf, "echo", 4) == 0) {
+                    /* echo command just echoes back the input string */
+                    Serial.write("ECHO:");
+                    for (int i = 4; i < count; i++) {
+                        Serial.write(buf[i]);
+                    }
+                } else if (strncmp(buf, "numb", 4) == 0) {
+                    /* numb command prints input number if various formats */
```

Comment:
typo: `if various formats` should be `in various formats`

Reply:
Thanks for spotting this! Fixed!